notebooks/data_analysis.ipynb

```diff
@@ -2,7 +2,45 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 18,
+   "id": "4aa55cde-75ba-4114-87af-a4e5748ddd1c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Downloading...\n",
+      "From: https://drive.google.com/uc?id=1V2XEyLiNBv2GsqG4SFUQVACqCUZ6h3w4\n",
+      "To: /home/jovyan/ohw21-proj-deep-currents/notebooks/dataset-duacs-rep-global-merged-allsat-phy-l4_1628139790857.nc\n",
+      "361MB [00:01, 232MB/s]  \n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "'dataset-duacs-rep-global-merged-allsat-phy-l4_1628139790857.nc'"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "try:\n",
+    "    import gdown\n",
+    "except:\n",
+    "    !pip install gdown\n",
```

Comment:
Using `!pip install` can [leave your environment in an unstable state](https://oceanhackweek.github.io/ohw-resources/prep/conda/#temporary-packages). If you use `%pip install` it has some tweaks to play better.